packages/proxy/src/proxy.ts

```diff
@@ -449,6 +453,10 @@ export async function proxyV1({
       }
       proxyResponseHeaders[name] = value;
     });
+    if (provider) {
+      setHeader(PROVIDER_HEADER, provider);
+      proxyResponseHeaders[PROVIDER_HEADER] = provider;
+    }
```

Comment:
i wonder if we should use the secret name, so that if you have eg multiple openai endponts, we can distinguish?

Reply:
oh yeah that's the value it's actually using, i'll name it better